ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/loopback_interfaces/overlay-loopback.j2

```diff
@@ -1 +1 @@
-EVPN_Overlay_Peering
+{% if switch.default_mpls_overlay_role in ["server", "client"] %}MPLS_Overlay_peering{% elif switch.mpls_lsr is arista.avd.defined(true) %}LSR_Router_ID{% else %}EVPN_Overlay_Peering{% endif %}
```

Comment:
Missing blank line at the end of file!

Reply:
There is a blank line on my local branch that i just pushed, maybe github is not seeing it for some reason...